drivers/FlashIAP.cpp

```diff
@@ -31,6 +33,8 @@ namespace mbed {
 
 SingletonPtr<PlatformMutex> FlashIAP::_mutex;
 
+#define BLANK 0xFF
```

Comment:
Please use more descriptive name (BLANK can come from targets code and can have diff value), `FLASHIAP_BLANK` or similar would be better